test/bdd/fixtures/edv-rest/.env

```diff
@@ -1,14 +1,14 @@
-  #
+#
 # Copyright SecureKey Technologies Inc. All Rights Reserved.
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-EDV_REST_IMAGE=docker.pkg.github.com/trustbloc/edv/edv-rest
-EDV_REST_IMAGE_TAG=0.1.4
+EDV_REST_IMAGE=docker.pkg.github.com/trustbloc-cicd/snapshot/edv-rest
+EDV_REST_IMAGE_TAG=0.1.5-snapshot-f2cf57a
 
 EDV_HOST=0.0.0.0
 EDV_PORT=8071
 EDV_DATABASE_TYPE=couchdb
 EDV_DATABASE_URL=couchdb.example.com:5984
-EDV_DATABASE_PREFIX=edv
+EDV_DATABASE_PREFIX=edv
```

Comment:
Looks like you accidentally removed the newline at the end of the file here.

Reply:
fixed!